### PR TITLE
docs: Documentation updates prior to release candidate 2.0 release

### PR DESCRIPTION
**Pull request by davidberenstein1957**
# Generic Pull Request Template

- Add trailing slash to facilitate better versioning
- push-latest for default docs config
- updated pull request and issue templates
- added `--pre` to install to allow for pre-release candidates
- Added a changelog to 2.0 and mentioned it in 1.0

**Type of change** 

- Improvement (change adding some improvement to an existing functionality)
- Documentation update

**How Has This Been Tested**

NA

**Checklist**

- [ ] I added relevant documentation
- [ ] follows the style guidelines of this project
- [ ] I did a self-review of my code
- [ ] I made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I filled out [the contributor form](https://tally.so/r/n9XrxK) (see text above)
- [ ] I have added relevant notes to the CHANGELOG.md file (See https://keepachangelog.com/)

### CODE REVIEW

File: argilla/mkdocs.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Project information
 site_name: Argilla Docs
-site_url: https://argilla-io.github.io/argilla-python
+site_url: https://argilla-io.github.io/argilla-python/
```
Comment (frascuchon) [inline argilla/mkdocs.yml]: This should be https://argilla-io.github.io/argilla/, right?

Comment (davidberenstein1957) [reply argilla/mkdocs.yml]: yes, good catch

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,46 +1,28 @@
...
-- [ ] I did a self-review of my code
-- [ ] I made corresponding changes to the documentation
-- [ ] My changes generate no new warnings
-- [ ] I have added tests that prove my fix is effective or that my feature works
-- [ ] I filled out [the contributor form](https://tally.so/r/n9XrxK) (see text above)
-- [ ] I have added relevant notes to the CHANGELOG.md file (See https://keepachangelog.com/)
\ No newline at end of file
+<!--  Please go over the checklist and make sure you've taken everything into account -->
+
+- I added relevant documentation
```
Comment (sdiazlor) [inline .github/pull_request_template.md:22]: In fact, it's not a checklist

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,46 +1,28 @@
...
+- Improvement (change adding some improvement to an existing functionality)
+- Documentation update
 
 **How Has This Been Tested**
-
-(Please describe the tests that you ran to verify your changes. And ideally, reference `tests`)
-
-- [ ] Test A
-- [ ] Test B
+<!--  Please delete options that are not relevant. Remember to title the PR according to the type of change  -->
```
Comment (sdiazlor) [inline .github/pull_request_template.md]: Shouldn't this comment go above "How has this been tested"?

File: argilla/README.md [hunk 4]
```diff
@@ -9,9 +9,8 @@ The argilla-sdk is an experimental project to rebuild Argilla's python client in
 
 Since this package is not yet published on PyPi, you can install it directly from the repository:
 
-# TODO: Changed this to use the testpypi releases.
 ```console
-pip install git+https://github.com/argilla-io/argilla-python.git
```
Comment (sdiazlor) [inline argilla/README.md:14]: It should also be changed in installation.md and quickstart.md

**Review by sdiazlor — COMMENTED** [review]
Some small comments

File: argilla/README.md (hunk 4 above)
Comment (sdiazlor) [reply argilla/README.md:14]: @davidberenstein1957 In installation.md, the change is still missing.

**Review by frascuchon — APPROVED** [review]

**Pull request merged**